### PR TITLE
New Mu Microsite

### PR DESCRIPTION
**Pull request by juanpedromoreno**
This PR brings the new microsite settings for `higherkindness`.

Additionally, updates the Scala 2.12 version and the sbt version.

### CODE REVIEW

**Comment by calvellido** [comment]
7d5cb2a should complete the needed changes @juanpedromoreno

File: docs/src/main/tut/annotations.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
 title: Annotations
-permalink: /docs/rpc/annotations
+permalink: /annotations
```
Comment (fedefernandez) [inline docs/src/main/tut/annotations.md]: On root? should it be `/mu/scala/annotations`?

**Comment by juanpedromoreno** [comment]
Thank you @calvellido , looks great to me!

File: docs/src/main/tut/annotations.md (hunk 1 above)
Comment (calvellido) [reply docs/src/main/tut/annotations.md]: That is set through the `urlBase` parameter

Comment (juanpedromoreno) [reply docs/src/main/tut/annotations.md]: It's ok, the baseUrl is doing the rest ;)

**Pull request merged**